### ISSUE TITLE
About parameters

### ISSUE THREAD

**Issue by wuhanfeng-000** [XiongjunGuan/JIPNet#5]
Thank you for open this great job, i have some questions:
1.In the chart about model size in the last paragraph of the article, it shows that the model is 25.7M, but when I counted the parameters using code, it came out to 36.65M. I'm not sure which side is incorrect. Thank you.Code is here, config_path="./ckpts/JIPNet/config.yaml" is:
   trainable_params = sum(p.numel() for p in model.parameters() if p.requires_grad)
    print(f"Trainable parameters: {trainable_params / 1024 / 1024} MB")
2.In the application of fingerprint recognition, it is often necessary to extract features in advance. According to the model described, for a single 160x160 (25KB) image, the extracted features would be 264x20x20xsizeof(float)/1024=412.5KB, which is almost a 16-fold increase. Have you tried reducing the mid_dim(264) or splitting it into 264x10x10? I do not have the training code, so I am not sure whether such a reduction would lead to catastrophic consequences.
3.I have some unfortunate news to share. When I directly applied the inference model to our own dataset, the performance was not particularly impressive, as it yielded relatively high scores for many different fingers. However, this does not detract from the fact that this is still a commendable piece of work.

**Comment by XiongjunGuan** [comment]
About your questions:
1. We only consider the backbone and recognition branch when calculating the model size. Because in applications, when relative pose information is not specifically required, another pose branch is unnecessary.
2. I have not made this attempt. 
The primary objective of this work is to address the challenge of classifying difficult-to-distinguish data. For cases involving similar poses, as you mentioned, employing minutiae or fixed-length descriptors (which are usually space-efficient) can yield satisfactory matching results. However, in more complex scenarios, excessive feature compression may compromise the accuracy of verification.
In summary, I believe this method is best suited for applications where matching precision is critical, one-to-one comparisons are feasible (without the need to store features), and minimizing false rejections is a priority. Alternatively, it could be used in conjunction with other techniques to perform an initial screening, followed by applying this method to ambiguous or doubtful samples for further verification.
Reducing feature occupancy is a topic worthy of further exploration and is of great practical value.
3. I guess it may be caused by modality differences. Our model is trained using data simulated by the FVC data. Although some enhancement algorithms are used in the feature extraction training stage to enhance modality robustness as much as possible, it is still not as intuitive as the effect of fine-tuning.
